cypress/integration/settings/circulation/general/edit-staff-slips.spec.js

```diff
@@ -0,0 +1,50 @@
+import TestType from '../../../../support/dictionary/testTypes';
+import EditStaffClips from '../../../../support/fragments/circulation/editStaffClips';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+// TO DO: update test with duplicate and edit methods, after PO will review test case.
+describe('ui-circulation-settings: Edit Staff slip settings', () => {
+  const editStaffClipsHold = { ...EditStaffClips.defaultUiEditStaffClips };
+  beforeEach('login', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(`${SettingsMenu.circulationStaffSlipsPath}`);
+  });
+
+  it('C347901 Staff slip settings', { tags: [TestType.smoke] }, () => {
+    EditStaffClips.editHold(editStaffClipsHold);
+    EditStaffClips.fillStaffClips();
```

Comment:
Probably you can wrap fill-save-preview-check into one method? Since it's repeating again and again